### PR TITLE
prod-host-setup: add kvm-pit recommendations

### PR DESCRIPTION
**Pull request by alindima**
# Reason for This PR

#2777 

## Description of Changes

- [ ] This functionality can be added in [`rust-vmm`][1].

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license.

## PR Checklist

`[Author TODO: Meet these criteria.]`
`[Reviewer TODO: Verify that these criteria are met. Request changes if not]`

- [x] All commits in this PR are signed (`git commit -s`).
- [x] The issue which led to this PR has a clear conclusion.
- [x] This PR follows the solution outlined in the related issue.
- [x] The description of changes is clear and encompassing.
- [x] Any required documentation changes (code and docs) are included in this PR.
- [x] Any newly added `unsafe` code is properly documented.
- [x] Any API changes follow the [Runbook for Firecracker API changes][2].
- [x] Any user-facing changes are mentioned in `CHANGELOG.md`.
- [x] All added/changed functionality is tested.

[1]: https://github.com/rust-vmm
[2]: ../docs/api-change-runbook.md

### CODE REVIEW

File: docs/prod-host-setup.md [hunk 1]
```diff
@@ -157,6 +157,36 @@ Additional details of Jailer features can be found in the
...
+### Constrain CPU overhead caused by kvm-pit kernel threads
+
+The current implementation results in host CPU usage increase on x86 CPUs when
+a guest injects timer interrupts with the help of kvm-pit kernel thread.
+kvm-pit kthread is by default part of the root cgroup.
+
+To mitigate the CPU overhead we recommend two system level configurations.
+
+1.
+    Use an external agent to move the kvm-pit/ kernel thread in the microVM’s
```
Comment (AlexandruCihodaru) [inline docs/prod-host-setup.md:169]: nit: Maybe we should use here the full name format namely: kvm-pit/<pid of firecracker process>

Comment (alindima) [reply docs/prod-host-setup.md]: done

**Review by AlexandruCihodaru — APPROVED** [review]
🚢

**Review by dianpopa — APPROVED** [review]

File: docs/prod-host-setup.md [hunk 2]
```diff
@@ -157,6 +157,36 @@ Additional details of Jailer features can be found in the
...
+
+The current implementation results in host CPU usage increase on x86 CPUs when
+a guest injects timer interrupts with the help of kvm-pit kernel thread.
+kvm-pit kthread is by default part of the root cgroup.
+
+To mitigate the CPU overhead we recommend two system level configurations.
+
+1.
+    Use an external agent to move the `kvm-pit/<pid of firecracker>` kernel
+    thread in the microVM’s cgroup (e.g., created by the Jailer).
```
Comment (gregorybrzeski) [inline docs/prod-host-setup.md:170]: nit: maybe it should be more specific like: "kernel thread from root cgroup to the microVM's cgroup"

**Pull request merged**